sanowret-crystal.lic

```diff
@@ -0,0 +1,39 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+no_kill_all
+no_pause_all
```

Comment:
I _think_ this means that if you are arrested, `jail-buddy` won't be able to pause this script. I don't see any error handling below for the case in which you've temporarily lost your crystal.